osu.Game/Overlays/Music/PlaylistOverlay.cs

```diff
@@ -89,6 +92,17 @@ private void load(OsuGameBase game, BeatmapDatabase beatmaps, OsuColour colours,
             };
         }
 
+        protected override void Dispose(bool isDisposing)
+        {
+            if (beatmaps != null)

```

Comment:
these shouldn't be needed since we are instantiating the `PlaylistList` ourselves.